### PR TITLE
[docs] per-request sampling params

### PR DESCRIPTION
**Pull request by stevhliu**
adds docs for #45026

### CODE REVIEW

File: docs/source/en/continuous_batching.md [hunk 1]
```diff
@@ -124,6 +124,18 @@ Cancel a request with [`~ContinuousBatchingManager.cancel_request`].
 manager.cancel_request(request_id="my_request")
 ```
 
+### Per-request sampling parameters
+
+Enable `per_request_processors` to apply `temperature`, `top_k`, and `top_p` independently per request within the same forward pass to allow different sampling parameters for different requests (creative, high-temperature outputs versus precise, low-temperature ones for example).
```
Comment (remi-or) [inline docs/source/en/continuous_batching.md:129]:
```suggestion
Enable `per_request_processors` to apply `temperature`, `top_k`, and `top_p` independently per request within the same forward pass to allow different sampling parameters for different requests (creative, high-temperature outputs versus precise, low-temperature ones for example).
Each parameter must have a non-default value in the `GenerationConfig` to guarantee the associated logits processor is created at runtime. For instance, if per-request temperatures are needed, then the `GenerationConfig` temperature should not be `None` or `1`. Requests with temperatures of 1 can still be created afterwards.

**Review by remi-or — APPROVED** [review]
LGTM! I added a small technical note on a current limitation.
